objects_tdd/spec/lib/television_spec.rb

```diff
@@ -0,0 +1,108 @@
+require './lib/television'
+
+describe Television do
+  it 'can be created' do
+    expect(subject).to be_a(Television)
+  end
+
+  describe '#on?' do
+    it 'returns false by default' do
+      expect(subject.on?).to eq(false)
+    end
+  end
+
+  describe '#power' do
+    it 'toggles power on and off' do
+      subject.power
+      expect(subject.on?).to eq(true)
+      subject.power
+      expect(subject.on?).to eq(false)
+    end
+  end
+
+  describe '#set_channel(number)' do
+    it 'sets the channel to the number if within valid range' do
+      subject.power
+      subject.channel_set(45)
+      expect(subject.channel).to eq(45)
+    end
+
+    it 'leaves the channel unchanged if outside valid range' do
+      subject.power
+      subject.channel_set(874)
+      expect(subject.channel).to eq(3)
+    end
+  end
+
+  describe '#channel_up' do
+    it 'increments the channel by one' do
+      subject.power
+      subject.channel_up
+      expect(subject.channel).to eq(4)
+    end
```

Comment:
What if you hit `channel_up` and you're already on the highest channel?  Write a test that verifies it "rolls" over to the lowest channel.